### DISCUSSION TITLE
I wanna commit translation to this project

### DISCUSSION THREAD

**Discussion by vuongtuha** [filecxx/FileCentipede#410] [Q&A]
I finished translating your text into my mother-tongue language. And I exported it to file *.json on my PC. I wonder how could my translate appear to everyone using it later?
Sorry, I don't have technical background and don't know how to chat in Github. And hope you could guide me more clearly. Thx!

**Comment by filecxx** [comment]
Thanks for the translation.

Please drop the file into the text editor.

(Attach files by dragging & dropping, selecting or pasting them.).

Then, I'll do the rest.

**Comment by vuongtuha** [comment]
Here's my translation to vi_VN language. 
https://github.com/vuongtuha/pontoon/blob/main/vi_VN.lang
However, there are still some details that I am not satisfied with.
I'm still using "my language" version on my daily workload. And I would like to tune it more clearly and easy to understand. Hope that more ppl will be using your application, and improve my translation, too. More heads are better than one!

**Comment by filecxx** [comment]
The next version release time may be in December or later, I'll add the vi_VN.lang file.

Thank you very much, if you need anything, please let me know.

**Comment by vuongtuha** [comment]
Don't mind, I'm working as a freelancer and also as a Mozilla Pontoon contributor. It is common for me to use rolling release software.
So on, if there's anything new update, I can realize it soon. Or at least, GitHub will notify me via email 😉
At this time, I can adjust the translation to my best. And if you need a pre-release translation, you would feel free to send me lang text file. My TG: @vuongtuha
Regards and good luck!

**Comment by filecxx** [comment]
I'm currently making a new script language and a multi-protocol instant messaging client like [Pidgin](https://www.pidgin.im/), and one of the features is chatting through GitHub or email, I believe it could solve this problem.
